community/dbms/src/main/java/org/neo4j/commandline/dbms/ImportCommand.java

```diff
@@ -32,18 +31,51 @@
 import org.neo4j.server.configuration.ConfigLoader;
 
 public class ImportCommand implements AdminCommand
 {
     public static final String DEFAULT_REPORT_FILE_NAME = "import.report";
+    private static final String[] allowedModes = {"database", "csv"};
+    private static final Arguments arguments = new Arguments()
+            .withDatabase()
+            .withAdditionalConfig()
+            .withArgument( new OptionalNamedArg( "mode", allowedModes, "csv",
+                    "Import a collection of CSV files or a pre-3.0 installation." ) )
+            .withArgument( new OptionalNamedArg( "from", "source-directory", "",
+                    "The location of the pre-3.0 database (e.g. <neo4j-root>/data/graph.db)." ) )
+            .withArgument( new OptionalNamedArg( "report-file", "filename", ImportCommand.DEFAULT_REPORT_FILE_NAME,
+                    "File in which to store the report of the import." ) )
+            .withArgument( new OptionalNamedArg( "nodes[:Label1:Label2]", "\"file1,file2,...\"", "",
+                    "Node CSV header and data. Multiple files will be logically seen as " +
+                            "one big file from the perspective of the importer. The first line " +
+                            "must contain the header. Multiple data sources like these can be " +
+                            "specified in one import, where each data source has its own header. " +
+                            "Note that file groups must be enclosed in quotation marks." ) )
+            .withArgument( new OptionalNamedArg( "relationships[:RELATIONSHIP_TYPE]", "\"file1,file2,...\"", "",
+                    "Relationship CSV header and data. Multiple files will be logically " +
+                            "seen as one big file from the perspective of the importer. The first " +
+                            "line must contain the header. Multiple data sources like these can be " +
+                            "specified in one import, where each data source has its own header. " +
+                            "Note that file groups must be enclosed in quotation marks." ) )
+            .withArgument( new OptionalNamedArg( "id-type", new String[]{"STRING", "INTEGER", "ACTUAL"},
+                    "STRING", "Each node must provide a unique id. This is used to find the correct " +
+                    "nodes when creating relationships. Possible values are " +
+                    "STRING: arbitrary strings for identifying nodes, " +
+                    "INTEGER: arbitrary integer values for identifying nodes, " +
+                    "ACTUAL: (advanced) actual node ids. " +
+                    "For more information on id handling, please see the Neo4j Manual: " +
+                    "http://neo4j.com/docs/operations-manual/current/deployment/#import-tool" ) )
+            .withArgument( new OptionalNamedArg( "input-encoding", "character-set", "UTF-8",
+                    "Character set that input data is encoded in." ) );
```

Comment:
I'm worried that it'll be hard to figure out how to use this one, particularly the `database` mode, because it's very simple but that's not obvious. Maybe for this one case it would be good to put a couple of simple examples in the description text.